proxy/RegressionSM.cc

```diff
@@ -227,16 +223,12 @@ RegressionSM::RegressionSM(const RegressionSM &ao) : Continuation(ao)
 {
   RegressionSM &o = *(RegressionSM *)&ao;
 
-  t         = o.t;
-  status    = o.status;
-  pstatus   = o.pstatus;
-  parent    = &o;
-  nwaiting  = o.nwaiting;
-  nchildren = o.nchildren;
-
-  for (intptr_t i = 0; i < nchildren; i++) {
-    children(i) = o.children[i]->clone();
-  }
+  t        = o.t;
+  status   = o.status;
+  pstatus  = o.pstatus;
+  parent   = &o;
+  nwaiting = o.nwaiting;
+  children = o.children;
```

Comment:
Does this assignment call `clone`, or did you add a copy constructor to handle that?